graphql/schema/types/metadata.graphql

```diff
@@ -85,6 +85,8 @@ input ScanMetadataInput {
   scanGeneratePhashes: Boolean
   """Generate image thumbnails during scan"""
   scanGenerateThumbnails: Boolean
+  """Detect scene captions"""
+  scanDetectCaptions: Boolean
```

Comment:
What is the benefit of making this optional?

Reply:
This wasn't intentional. I can update it to be required.

Reply:
Sorry, what I mean is - is there any reason not to have the scan task always detect captions?

Reply:
The detect captions toggle was introduced when the code was still looking for captions by trying various name combinations to see if we could get a match. That approach was expensive, which led to this toggle being added so users without captions wouldn't be impacted by that search when it didn't benefit them. The toggle could probably be removed now that the search code is more intelligent.